src/types/types.js

```diff
@@ -398,7 +398,7 @@ export type EdgeTransaction = {
 
   // Core:
   metadata?: EdgeMetadata,
-  wallet?: EdgeCurrencyWallet, // eslint-disable-line no-use-before-define
```

Comment:
I think we need to add the block at the end of the interface:

```
// Deprecated:
wallet?: EdgeCurrencyWallet // We never populate this
```

That way, it's not a breaking change.